### DISCUSSION TITLE
More group subfolders in my collection?

### DISCUSSION THREAD

**Discussion by DIrgi** [devemux86/cruiser#513] [Q&A] [answered]
1st of all sorry if this topic exist already somewhere but I didn't find it via the search.

I ask/request if it's possible to have at least one more subfolder level in the several "my collection" track/poi/route groups.
Definitely for better structuring. If needed I can give example, why.

**Comment by devemux86** [comment]
@DIrgi thanks for the suggestion.

We could think about it, however it is much more complicated to manage, also in the user interface.

**Reply by DIrgi** [reply]
Normally I would say, well, one folder level is enough but....
I just recognized at least one missing level.

Just an example (with POIs)
![image](https://github.com/devemux86/cruiser/assets/57968526/5179037f-fc1a-4f55-88cb-ade13e2d417c)

So far, I can just create one group(folder)level in this case "Pass group","Motorbike repair"....and so on.
The next sublevel cannot have any groupfolders, so finally all passes from France, Germany, Switzerland....will be in one Group. This is a huge list, as well activating/disactivating (in this way country wise(by group)) is a night mare
I hope it's clear what I mean and where I see a need.

**Reply by devemux86** [reply, marked as answer]
Can also use names like "Group 1 | Group 2 | ...", e.g. "Pass | France", "Pass | Germany".
Can sort or reorder them and also can (de) activate whole groups easily with one click.

Tree structures are complicated in storage, user interface, etc. so this is not a decision taken lightly. :slightly_smiling_face:

**Comment by DIrgi** [comment]
> Can also use names like "Group 1 | Group 2 | ...", e.g. "Pass | France", "Pass | Germany". Can sort or reorder them and also can (de) activate whole groups easily with one click.

Not Really because then the 1st level list is very long.
i.e.

POIs:
- PassesFR
- PassesDE
- PassesCH
- PassesAT
- PassesES
- PassesPT
.....
.....
- CocktailBars
- MiniBars
- BigBars
- SmallBars
....
....
these are just examples.
So coming back just to the passes. If I want to see them all, I have to click on each Group folder (which can be some)

> Tree structures are complicated in storage, user interface, etc. so this is not a decision taken lightly. 🙂

Hmm, I don't know how software programming work in this case but.... some thought back....
The structure need not to be 1:1 on the android system. It can be just  stored in one folder (on the android OS) but.....
the (virtual) structure shown in the cruiser menu is just stored within a cruiser (whatever) database or settings file.
So cruiser is the interface (the path keeper) or file manager between what his displays in a data structure and the real file folder structure on Android.

**Comment by DIrgi** [comment]
I do a re-warming of this topic - now for cruiser 5.
Is it possible to have some more subfolders in cruiser 5 at least one level?

See example, this is what I have now under the favorites level (and it's just an extraction)

![image](https://github.com/user-attachments/assets/6a39e4c3-c8b7-496d-8215-c88736d039e4)

You see i.e. a lot of Pässe(passes). 
Lots of them (because there are so many in my list) contain already some related single POIs.
Since they are all Passes, I would like to have them in a subfolder which I can call i.e. "Passes" 
and give them all the same symbol - now I have to switch it for every entry and the list is very long 
next to other POIs and rather confusing and as well structures as it can be under the folder limit aspect.

btw. I switch them on/off when I need them - I don't have them all together visible at the same time.

**Reply by devemux86** [reply]
> Is it possible to have some more subfolders in cruiser 5 at least one level?

It is more complicated, also in the interface, we could see it in the future.

**Reply by DIrgi** [reply]
Hmmm, I know this statement already 😉
I hoped it will be better and easier with the new Cruiser engine/version.

**Reply by devemux86** [reply]
This is not related to the map engine. The map view is just to view the map.
Everything else is done on top with other Cruiser libraries. :slightly_smiling_face:
